package.json

```diff
@@ -4,7 +4,10 @@
     "description": "",
     "scripts": {
         "start": "webpack --mode development",
-        "dev": "webpack --mode development --watch"
+        "watch": "webpack --mode development --watch",
+        "dev": "webpack --mode development",
```

Comment:
is there a reason to have both `start` and `dev`, or could `start` be removed now? also, i like the 1:1 match with our dev, beta, prod scheme :+1:

Reply:
Well I set up vizlab to have `start` as the default, so by keeping it, you can still run `vizmake()` without any errors. I could delete `dev` since it is the same, but I added it so we could have that 1:1 match. Let me know if you have other ideas though!

Reply:
Ah, right. Well, I could see switching the default in vizmake() to be `dev`, but that seems like a lot of work...I'm OK just leaving as-is.